src/microbe_stage/MicrobeStage.tscn

```diff
@@ -427,6 +428,12 @@ script = ExtResource( 7 )
 margin_left = -0.943848
 margin_right = -0.943848
 
+[node name="ScreenFilter" parent="." instance=ExtResource( 23 )]
```

Comment:
Okay so the screen filter only works in the microbe stage? Is there a reason why this wasn't made with the overlay system to apply to the entire game?